content/documentation/pages/6-recipes/1-polyglot/2-polyglot-python-task.md

```diff
@@ -40,60 +71,66 @@ try:
     if get_cmd_arg('error.message') is not None:
         raise Exception(get_cmd_arg('error.message'))
 
+    # Set task's status to COMPLETED.
     status.completed()
 
 except Exception as exp:
-    # set status to FAILED
+    # Set task's status to FAILED.
     status.failed(1, 'Task failed: {}'.format(exp))
 ```
 
-If the `--error.message=<SomeText>` is added as a launch argument the pyton_task.py will imitate an execution failure.
+<!--IMPORTANT-->
 
-The TaskStatus class inside [task_status.py](https://github.com/tzolov/scdf-polyglot-experiments/blob/master/python_task_with_status/util/task_status.py) utility can update the content of the `TASK_EXECUTION` table to emulate the status life cycle changes.
-The TaskStatus is initialized with the `task id` and `sqlalchemy url` computed from the command arguments with the help of the `task_args.py`.
-The TaskStatus can set the task status to `running`, `completed` or `failed(with exitCode, errorMessage)`.
+Unlike the `Spring Cloud Tasks`, the Python tasks are responsible to manage their own states within the SCDF database.
```

Comment:
`Spring Cloud Task`

Reply:
Maybe:

> Since the Python script is not managed by `Spring Cloud Task`, is the user's responsibility to manage and update the progress with the SCDF database.